taca_ngi_pipeline/deliver/deliver_dds.py

```diff
@@ -478,12 +479,8 @@ def _set_project_details(self, projectid, given_desc=None):
         if not self.project_desc:
             try:
                 prj_order = self._get_order_detail()
```

Comment:
Do we need to fetch the order details here then? We could just use self.projectname instead.

Reply:
`projectname` seems to be an invalid query

Reply:
Yes, I don't think the project name is set anywhere in the DDSProjectDeliverer object so it needs to be fetched, like Chuan has done here 👍

Reply:
`DDSProjectDeliverer` inherits from `ProjectDeliverer` which inherits from `Deliverer` which has projectname here https://github.com/SciLifeLab/taca-ngi-pipeline/blob/d577b9387e3bbf72350983ca9c12e7a2ddcd9653/taca_ngi_pipeline/deliver/deliver.py#L95

Does it not work then?

Reply:
No, `name` and `uppnex_id` (line 103) are old keys, and I don't think they exist anymore in our database.

Reply:
@aanil Hi Anand, please hold on with this PR. I will test more on it. Thanks!